### PR TITLE
simple attempt at shot on the move

### PR DESCRIPTION
**Pull request by bengrif**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -246,12 +246,16 @@ private void configureDriverBindings() {
...
+            new ParallelCommandGroup(
+                new AutoAimCommand(
+                        driveSubsystem,
+                        FieldPositions.getInstance().getSpeakerTranslation(),
+                        () -> getJoystickInput(driverController, JoystickAxis.Ly),
+                        () -> getJoystickInput(driverController, JoystickAxis.Lx))
+                    .repeatedly(),
+                manipulatorSubsystem
+                    .getShooterAndArmPrepareCommand(() -> getDistanceToSpeaker())
+                    .repeatedly()));
```
Comment (Kellenn) [inline src/main/java/frc/robot/RobotContainer.java:258]: Let's use method chaining where we can.

```suggestion
            new AutoAimCommand(
                    driveSubsystem,
                    FieldPositions.getInstance().getSpeakerTranslation(),
                    () -> getJoystickInput(driverController, JoystickAxis.Ly),
                    () -> getJoystickInput(driverController, JoystickAxis.Lx))
                .repeatedly()
                .alongWith(
                    manipulatorSubsystem
                        .getShooterAndArmPrepareCommand(this::getDistanceToSpeaker)
                        .repeatedly()));
```

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -308,6 +312,11 @@ private void configureDriverBindings() {
             Commands.runOnce(() -> driverController.getHID().setRumble(RumbleType.kBothRumble, 0)));
   }
 
+  public double getDistanceToSpeaker() {
+    return distanceFilter.calculate(
+        LocationHelper.getDistanceToPose(
+            driveSubsystem.getPose(), FieldPositions.getInstance().getSpeakerCenter()));
+  }
```
Comment (Kellenn) [inline src/main/java/frc/robot/RobotContainer.java:319]: Can we put this logic in a subsystem?

**Comment by bengrif** [comment]
Tested it and it wasn't performant enough to make a difference.

**Comment by bengrif** [comment]
Doesn't work well enough for the drivers.

**Pull request closed without merging**